federation/cmd/federated-controller-manager/OWNERS

```diff
@@ -0,0 +1,4 @@
+assignees:
```

Comment:
`federation-controller-manager` seems like a better name than `federated-controller-manager`.
(We should update apiserver's name as well)